externs/browser/html5.js

```diff
@@ -362,6 +362,17 @@ Path2D.prototype.arcTo = function(x1, y1, x2, y2, radius) {};
  */
 Path2D.prototype.rect = function(x, y, w, h) {};
 
+/**
+ * @param {number} x
+ * @param {number} y
+ * @param {number} w
+ * @param {number} h
+ * @param {(number|DOMPointInit|Array<DOMPointInit>)=} radii
+ * @return {undefined}
+ * @override
```

Comment:
What is this overriding?

Reply:
`CanvasPathMethods` updated.  This now follows the pattern of all of the other inherited methods.